subsys/mpsl/fem/Kconfig

```diff
@@ -143,10 +143,6 @@ config MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL
 
 endif   # (MPSL_FEM_NRF21540_GPIO || MPSL_FEM_NRF21540_GPIO_SPI) && MPSL_FEM
 
-config MPSL_FEM_POWER_VOLTAGE
-	int "Voltage which is used to power the FEM chip in mV"
-	default 3000
-
 config MPSL_FEM_POWER_MODEL
 	bool "Use a model to split the requested TX power into values used to control the internal state of the FEM"
 	default n
```

Comment:
If this is n by default then the GPIO SPI built in model wont be turned on by default, as the `MPSL_FEM_POWER_MODEL_CHOICE` won't even be defined

Reply:
`MPSL_FEM_POWER_MODEL` is selected by the GPIO/SPI variant, which overrides the default.

Reply:
Using `select` is not a good idea in my opinion. It will disallow the user setting turning the model of if he wishes. It will cause an error if the user attempts to set `CONFIG_MPSL_FEM_POWER_MODEL=n`

Reply:
> It will disallow the user turning the model off if he wishes

I agree, it makes `select` a poor choice. I think `imply` could be the correct choice here since it doesn't force a value. I've just verified Kconfig behavior with `select` replaced with `imply` and while the default is `CONFIG_MPSL_FEM_POWER_MODEL=y`, it's possible to turn it off both via command line and menuconfig.